### DISCUSSION TITLE
Discuss about coffee

### DISCUSSION THREAD

**Discussion by Motokuma** [NAIST-SE/AttractingContributors#1037] [2024 Attraction Contributions]
I agree the more acidic the coffee, the better.
Answer 'Yes' if you agree or 'No' if you disagree.

**Comment by kanariari3** [comment]
No.
I like non acidic coffee.

**Comment by gill876** [comment]
No, I prefer low acidity and sourness, with a lot of body!

Please contribute to my project!
https://github.com/gill876/AttractingContributors-Hiking
